### ISSUE TITLE
Nanosuit Boots of the Traveller does not absorb damage if fall distance is too high

### ISSUE THREAD

**Issue by pokeball99** [GTNewHorizons/GT-New-Horizons-Modpack#13419]
### Your GTNH Discord Username

Kekel7#9661

### Your Pack Version

2.3.0

### Your Server

Private

### Java Version

Other versions in between

### Type of Server

Don't know

### Your Expectation

To Not Take Damage while wearing Nanosuit Boots Of The Traveler

### The Reality

Took Damage when wearing a Jetpack not on Hover mode while wearing Nanosuit Boots Of The Traveler(i have not tested if this happens on other kinda of boots of the traveller but suspect this to be the case, maybe also related to #11772?)

### Your Proposal

to not take damage

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [X] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by chochem** [comment]
does not happen for me

**Comment by chochem** [comment]
did some more testing, absolutely zero damage.

**Comment by chochem** [comment]
see however https://discord.com/channels/181078474394566657/401118216228831252/1103427659993714869
so if the height is like 71+ and the durability is insufficient then you will take the full damage

**Comment by firenoo** [comment]
TL;DR each electric boots (electric, nano, quantum) first calculates the amount of energy it would take to absorb the damage. If the energy amount > energy in the boots, you take the full fall damage (which is probably fatal if your boots were full charge). For electric, this is ~30-40 blocks. For nano, this is ~66-75 blocks. For quantum this is ~165-175 blocks
Reference code:
https://github.com/GTNewHorizons/Electro-Magic-Tools/blob/master/src/main/java/emt/item/armor/boots/ItemElectricBootsTraveller.java#L192

A suggestion would be to have the boots take all the damage it can before passing the remaining damage to the player instead of skipping outright.
